api/v1alpha1/authorization_types.go

```diff
@@ -0,0 +1,47 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+// Authorization defines the authorization configuration.
```

Comment:
what would deny all requests look like ?
```
authorization: {}
```
or
```
authorization:
  rules:
  - action: Deny
```

Reply:
it could like
```
authorization:
  rules:
  - action: Deny
```